tests/X86/SSE/SQRTSS.S

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2017 Trail of Bits, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+TEST_BEGIN(SQRTSSv128v128, 1)
+TEST_INPUTS(
+    0x41100000,   // 9.0, a "perfect square," of which sqrt() is an integer
+    0x00000000,   // sqrt(0) should be 0
```

Comment:
I suggest adding in negative zero here too.

Reply:
Easier would be to define a macro with all these SQRT inputs, like:
```c++
#defne SQRT_INPUTS_32 \
  TEST_INPUTS( \
    ...)
```
Then just put `SQRT_INPUTS_32` in each square root test taking 32-bit arguments :-)